### PR TITLE
Add --log-scope for logging all crates

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
Right now we only log scoped to the current library, which may be not
enough especially when it comes to debugging issues related to
h2/grpc/tokio.

We now add a `--log-scope` flag, which can be used together with
`--log-level` to log all crates as well.
#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Added `--log-scope` flag to be able to log on all crates, not only the current lib.
```

### CODE REVIEW

File: src/config.rs [hunk 1]
```diff
@@ -5,15 +5,17 @@ use getset::{CopyGetters, Getters};
 use log::LevelFilter;
 use serde::{Deserialize, Serialize};
 use std::path::PathBuf;
+use strum::EnumString;
 
 #[derive(Builder, Clap, CopyGetters, Getters, Deserialize, Serialize)]
 #[builder(default, pattern = "owned", setter(into))]
 #[serde(rename_all = "kebab-case")]
 #[clap(
+    about("CRI - The Kubernetes Container Runtime written in Rust"),
```
Comment (Klaven) [inline src/config.rs:14]: as an example. this is not really part of this PR... I think this is fine in the current state of this repo (a very young hacking project). Just wanted to get your all thoughts/input so I can move forward and make my reviews better for you all.

**Review by Klaven — APPROVED** [review]
Been reviewing some of these PR's wondering what level of PR nitpickery we are looking for :) AKA, the update to the about section is unrelated but this is kind of in a hacking state. Anyhow. as I look to get involved I like to start with reviewing some PR's before I make one myself (my rust sucks anyhow)

Anyhow, I think this looks good and the addition too logging it awesome and a good addition.

File: src/config.rs (hunk 1 above)
Comment (saschagrunert) [reply src/config.rs:14]: Yeah, I agree. I admit being too lazy to open another PR for this change :innocent: 

The main reason to change this was to fix the comment in `cargo doc` while still preserving the line on the CLI output.

**Review by mrunalp — APPROVED** [review]

**Pull request merged**